src/main/scala/is/hail/asm4s/Code.scala

```diff
@@ -375,6 +404,12 @@ class CodeBoolean(val lhs: Code[Boolean]) extends AnyVal {
   def ||(rhs: Code[Boolean]): Code[Boolean] =
     lhs.toConditional || rhs.toConditional
 
+  def ceq(rhs: Code[Boolean]): Code[Boolean] =
+    lhs.toConditional.ceq(rhs.toConditional)
+
+  def cne(rhs: Code[Boolean]): Code[Boolean] =
+    lhs.toConditional.cne(rhs.toConditional)
```

Comment:
It's somewhat disappointing that something as simple as equality and inequality is two jumps in the JVM. I guess the JIT must tear this up into something faster though.